vmaas_sync/vmaas_sync.py

```diff
@@ -14,22 +14,22 @@
 
 from common.database_handler import DatabasePool, DatabasePoolConnection, NamedCursor
 from common.logging import init_logging, get_logger
-from common import mqueue
-from common.utils import vmaas_post_request
+from common import mqueue, utils, constants
 
 WEBSOCKET_RECONNECT_INTERVAL = 60
 LOGGER = get_logger(__name__)
 
 PROMETHEUS_PORT = os.getenv('PROMETHEUS_PORT', '8087')
 
-CANNOT_CNX = Counter('ve_vmaas_sync_cannot_cnx', '# of times failed to connect to VMaaS')
 REFRESH = Counter('ve_vmaas_sync_webscan_refreshes', '# of times VMaaS told us it had new data')
 CNX_FAIL = Counter('ve_vmaas_sync_websocket_failures', '# of times VMaaS websocket closed on us')
 CNX_RECONNECT = Counter('ve_vmaas_sync_websocket_recnx', '# of times attempted VMaaS websocket reconnect')
 
 VMAAS_HOST = os.getenv('VMAAS_HOST', 'http://vmaas_webapp:8080')
 VMAAS_CVES_ENDPOINT = "%s/api/v1/cves" % VMAAS_HOST
-ENABLE_RE_EVALUATION = os.getenv("ENABLE_RE_EVALUATION", "YES")
+VMAAS_REPOS_ENDPOINT = "%s/api/v1/repos" % VMAAS_HOST
+ENABLE_RE_EVALUATION = os.getenv("ENABLE_RE_EVALUATION", "YES") == "YES"
+ENABLE_REPO_BASED_RE_EVALUATION = os.getenv("ENABLE_REPO_BASED_RE_EVALUATION", "NO") == "YES"
```

Comment:
let's use `from distutils.util import strtobool` since it's used on other places already, it parses also other values than "YES" - True, true, yes, 1, on, etc.